addons/pos_hr/static/src/js/PaymentScreen.js

```diff
@@ -0,0 +1,18 @@
+odoo.define('pos_hr.PaymentScreen', function (require) {
+    'use strict';
+
+    const PaymentScreen = require('point_of_sale.PaymentScreen');
+    const Registries = require('point_of_sale.Registries');
+
+    const PosHrPaymentScreen = (PaymentScreen_) =>
+          class extends PaymentScreen_ {
+              async _finalizeValidation() {
+                  this.currentOrder.employee = this.env.pos.get_cashier();
+                  await super._finalizeValidation();
+              }
+          };
+
+    Registries.Component.extend(PaymentScreen, PosHrPaymentScreen);
+
+    return Chrome;
```

Comment:
Typo here.
The return doesn't really matter tbh because we never import the overridden component since the original one is overridden but for the moment to stay in line with the other files, return the original component
```suggestion
    return PaymentScreen;
```